### PR TITLE
Add test cases and examples of CostManagement.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

Add test cases and examples of CostManagement.

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/CostManagement/docs/Get-AzCostManagementExport.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+### Example 4: Get a cost management export by pipeline
+```powershell
+PS C:\> @{Id = 'subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f/providers/Microsoft.CostManagement/exports/TestExport' } | Get-AzCostManagementExport
+
+Name       Type
+----       ----
+TestExport Microsoft.CostManagement/exports
+```
+
+This command gets a cost management export by pipeline.
```
Comment (wyunchi-ms) [inline src/CostManagement/docs/Get-AzCostManagementExport.md:79]: Sorry, I think this example is not necessary for this cmdlet.

File: src/CostManagement/docs/Get-AzCostManagementExportExecutionHistory.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+This command gets execution history of cost management export.
+
+### Example 2: Get execution history of cost management export by object
+```powershell
+PS C:\> $export = Get-AzCostManagementExport -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f" -ExportName "ps-customcolum-t"
+PS C:\> Get-AzCostManagementExportExecutionHistory -InputObject $export
+
+Name Type
+---- ----
```
Comment (wyunchi-ms) [inline src/CostManagement/docs/Get-AzCostManagementExportExecutionHistory.md:48]: I think it's better that change this example as `by Piping`:
```powershell
PS C:\> Get-AzCostManagementExport -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f" -ExportName "ps-customcolum-t" | Get-AzCostManagementExportExecutionHistory
```
or
```powershell
PS C:\> $export = Get-AzCostManagementExport -Scope "subscriptions/9e223dbe-3399-4e19-88eb-0975f02ac87f" -ExportName "ps-customcolum-t"
PS C:\> $export | Get-AzCostManagementExportExecutionHistory
```
And please update this exmaple result after pulling the code.

File: src/CostManagement/docs/Get-AzCostManagementExportExecutionHistory.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -Scope
+The scope associated with query and export operations.
+This includes '/subscriptions/{subscriptionId}/' for subscription scope, '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}' for resourceGroup scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope and '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}' for Department scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/enrollmentAccounts/{enrollmentAccountId}' for EnrollmentAccount scope, '/providers/Microsoft.Management/managementGroups/{managementGroupId} for Management Group scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}' for billingProfile scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}/invoiceSections/{invoiceSectionId}' for invoiceSection scope, and '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/customers/{customerId}' specific for partners.
```
Comment (wyunchi-ms) [inline src/CostManagement/docs/Get-AzCostManagementExportExecutionHistory.md:103]: This is not accurate. `Scope` is not start with `/`. We need to change the description of all `Scope`.

File: src/CostManagement/docs/Get-AzCostManagementExportExecutionHistory.md (hunk 2 above)
Comment (LucasYao93) [reply src/CostManagement/docs/Get-AzCostManagementExportExecutionHistory.md]: Done

File: src/CostManagement/docs/Get-AzCostManagementExport.md (hunk 1 above)
Comment (LucasYao93) [reply src/CostManagement/docs/Get-AzCostManagementExport.md]: Done

File: src/CostManagement/docs/Get-AzCostManagementExportExecutionHistory.md (hunk 3 above)
Comment (LucasYao93) [reply src/CostManagement/docs/Get-AzCostManagementExportExecutionHistory.md:107]: This should be generated automatically. I shouldn't be able to change. isn't it ?

**Pull request merged**